lessons/PlottinginPythonwithMatplotlib/ricardo_harripaul_hwk2-2.py

```diff
@@ -0,0 +1,27 @@
+#import necessary modules for poly1d and arange from numpy
+from numpy import poly1d,arange
+import matplotlib
+
+#Generate an empty array for storing the roots
+roots=[]
+
+#Nested loops iterates through all the range of s and zeta passing the values
+#to the poly1d and solving the roots.  The roots are then stored in the roots array
+for s in arange(0.0,0.65,0.01):
+    for z in arange(0.0,0.62,0.01):
+        
+        p=poly1d([1,0,z-2,z,0],variable='s')
+        roots.append(p.r)
+ 
+#Generate empty array for storing non zero values of the roots   
+nonzeros=[]
+#This loop extracts out the non zero values of the roots and appends it to a new array
+for i in xrange(len(roots)-1):
```

Comment:
How do I add things to the new repo?  Do I just commit the changes in my
directory on my computer and push the changes to the fork?


On Wed, Aug 5, 2015 at 3:20 PM, Ricardo Harripaul <
ricardo.harripaul@gmail.com> wrote:

> So do I need to re add the repo to commit everything or should I just push?
>
>
> On Wed, Aug 5, 2015 at 3:08 PM, Elliott Sales de Andrade <
> notifications@github.com> wrote:
>
>> In lessons/PlottinginPythonwithMatplotlib/ricardo_harripaul_hwk2-2.py
>> <https://github.com/UofTCoders/studyGroup/pull/33#discussion_r36341552>:
>>
>> > +
>> > +#Generate an empty array for storing the roots
>> > +roots=[]
>> > +
>> > +#Nested loops iterates through all the range of s and zeta passing the values
>> > +#to the poly1d and solving the roots.  The roots are then stored in the roots array
>> > +for s in arange(0.0,0.65,0.01):
>> > +    for z in arange(0.0,0.62,0.01):
>> > +
>> > +        p=poly1d([1,0,z-2,z,0],variable='s')
>> > +        roots.append(p.r)
>> > +
>> > +#Generate empty array for storing non zero values of the roots
>> > +nonzeros=[]
>> > +#This loop extracts out the non zero values of the roots and appends it to a new array
>> > +for i in xrange(len(roots)-1):
>>
>> Use regular range for Python 3 compatibility. But actually, you don't
>> need i. You can just do for root in roots[:-1]: (but I'm not sure why
>> you're skipping the last one.)
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/UofTCoders/studyGroup/pull/33/files#r36341552>.
>>
>
>